tests/contrib/actions/jobs/test_tags.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from spidermon.contrib.actions.jobs.tags import JobTagsAction, AddJobTags, RemoveJobTags
+from spidermon.exceptions import NotConfigured
+
+TAG = "custom_tag"
+
+
+@pytest.mark.parametrize(
+    "tags, expected_tags", [(None, []), ([TAG], [TAG]), (TAG, [TAG])],
+)
+def test_job_tags_action_init(tags, expected_tags):
+    job_tags = JobTagsAction(tags)
+    assert job_tags.tags == expected_tags
+
+
+class JobTagsActionTests:
```

Comment:
Why the class here?

Reply:
The intention is to keep the code more organized, given the fact the all classes have methods with same name.
Do you suggest another approach?